test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -683,3 +683,24 @@ Describe "Pwsh exe resources tests" -Tag CI {
         $psversiontable.os | Should -MatchExactly "$($osversion.Major).$($osversion.Minor)"
     }
 }
+
+Describe 'Pwsh startup in directories that contain wild cards' -Tag CI {
+    BeforeAll {
+        $powershell = Join-Path -Path $PsHome -ChildPath "pwsh"
+        $dirnames = "[T]est","[Test","T][est","Test"
+        foreach ( $d in $dirnames ) { new-item -type Directory -path "${TESTDRIVE}/$d" }
+        $testcases = foreach ( $d in $dirnames ) { @{ Dirname = $d } }
```

Comment:
We iterate over `$dirname` twice, once for `new-item` and once for generating test cases. Should be do it in a single loop?

Reply:
sure